coverage_point_calculator/src/location.rs

```diff
@@ -0,0 +1,179 @@
+use rust_decimal::Decimal;
+use rust_decimal_macros::dec;
+
+use crate::RadioType;
+
+const RESTRICTIVE_MAX_DISTANCE: Meters = Meters(50);
+
+#[derive(Debug, Clone, PartialEq, PartialOrd)]
+pub struct Meters(u32);
```

Comment:
while this is not a hill I will die on, I do think it does add readability to the codebase.  It very obvious what the number means when reading through the code.

Reply:
I think it should only be a new struct if there are restrictions on the input type. Otherwise, just make it a type def:
```suggestion
pub type Meters = u32;
```

Reply:
I opted for a similar treatment to `latency_millis`. 
`Meters` has been demoted to a type alias, and the field has been renamed to `meters_to_asserted` which I think confronts a user of this struct with the same information as requiring them to provide a `Meters` struct.